thavalon-server/src/connections/account_handlers.rs

```diff
@@ -171,6 +173,22 @@ pub async fn handle_user_login(
     Ok(response)
 }
 
+/// Handles a user logging out. This revokes the user's refresh token.
+/// Currently this doesn't support "force logouts" of all devices.
```

Comment:
Force logout probably shouldn't be handled by log out - should either be on password update or just a button in account to log out of all devices.

Reply:
Fixed.